cdap-system-app-api/pom.xml

```diff
@@ -40,6 +40,11 @@
       <artifactId>cdap-storage-spi</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-proto</artifactId>
```

Comment:
We don't want to add cdap-proto as a dependency to an api module. See comment below